alienshow

```diff
@@ -0,0 +1,23 @@
+{
+  "slug": "alien show",
+  "title": "The Alien Show",
+  "description": "Meh Meh Meh",
+  "parent": "Comedy",
+  "parentYear": "2020",
+  "rating": "TV-PG",
+  "runtime": "4 seasons",
+  "labels": [
+    "animation",
+    "aliens",
+    "music"
+  ],
+  "credit": " Warner Bros. Television",
+  "altHero": "Aliens Rule",
+  "layout": "movie.pug",
+  "tags": [
+    "entry",
+    "TV series",
+    "comedy",
+    "new on nestflix"
+  ]
+}
```

Comment:
![ALIEN LOGO TITLE](https://github.com/user-attachments/assets/ec9014af-7959-40e5-877b-371df16cb031)

Reply:
![ALIEN SCREENCAP](https://github.com/user-attachments/assets/edac3004-9246-4732-bc40-b83c51a7e699)

Reply:
![THE ALIEN SHOW LOGO](https://github.com/user-attachments/assets/b4b87acb-12e4-464e-b202-03b98f40f488)